### ISSUE TITLE
playback cut off at end of song

### ISSUE THREAD

**Issue by violoncelloCH** [vixalien/muzika#207]
quite a lot of times when I'm listening to music, the playback cuts off some seconds before the end of the song, jumping to loading the next song...
thanks for the otherwise great app :)

**Comment by vixalien** [comment]
Yeah it seems to be some quirk with MPEG-DASH in GStreamer or the way Youtube Music serves up the MPEG streams. Nothing much I can do right now, except if one of the following 2 happen:

- GStreamer impements Webm DASH, see https://gstreamer.freedesktop.org/projects/dashwebm.html
- GJS fixes up it's GStreamer issues, allowing us to use the Stream Selection API instead of MPEG-DASH

In the meantime, you can get temporarily get rid of this issue for music files (not video) by selecting a different audio quality from "Auto".
